### ISSUE TITLE
Add SOCKS5 authentication directives

### ISSUE THREAD

**Issue by nemoyin** [dannote/socks-nginx-module#7]
rt

**Comment by nemoyin** [comment]
is there any directive?

**Comment by nemoyin** [comment]
and i want to use this module in openresty, how ?

**Comment by dannote** [comment]
No, there is no SOCKS authentication yet. To use it with OpenResty, build OpenResty from source and enable this module when configuring.

**Comment by gits7r** [comment]
I am interested to sponsor if you can implement support for authentication enabled socks5 proxies in the form of:

socks5://username:password@host:port/

(same as per industry standard)
Might be very useful.
